app/helpers/authorization.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+module Sinatra
+  module Authorization
+    def unauthorized!
```

Comment:
The `!` symbol is used to represent persistence or defin ite change. For example, it is used when saving to the db, `@user.save!` which means it saves to the db and returns the result or returns validation errors if it fails. It is also used to make persistent changes like `[1,4,5,2,6,9,2].sort!` permanently alters the arrangement of the values.


What's my point? I feel that the use of a bang here is being mis-represented. Maybe you could try something `def unauohorized_page` or `unauthorized_user` or `unauthorized_error_message`